app/javascript/packs/application.js

```diff
@@ -19,16 +19,25 @@ import ConditionalsController from "../src/controllers/conditionals-controller.j
 import ExpressionController from "../src/controllers/expression-controller.js";
 import ExpressionsController from "../src/controllers/expressions-controller.js";
 import SelectionRevealController from "../src/controllers/selection-reveal-controller.js";
+import OrderableItemsController from "../src/controllers/orderable-items-controller.js";
+import OrderableItemController from "../src/controllers/orderable-item-controller.js";
+import ComponentController from "../src/controllers/component-controller.js";
 
 Turbo.session.drive = false;
 window.Stimulus = Application.start();
+
+// Stimulus.debug = true;
```

Comment:
👀

Reply:
Yep - I only discovered this while doing this feature - its fairly handy - stops me having to drop in `console.log('x controller connected')` all over the place to work out the order of things!